src/utils/Utils.cpp

```diff
@@ -430,4 +432,18 @@ for x in range(256):
 			NRF_LOG_INFO("Done waiting");
 		#endif
 	}
+
+    void leaveValidation()
+    {
+        //ret_code_t err = sd_softdevice_disable();
```

Comment:
Remove this commented out code and add comments to explain how we use 2 bits to activate and de-activate validation mode